asv_bench/asv.conf.json

```diff
@@ -50,12 +50,13 @@
         "xlsxwriter": [],
         "xlrd": [],
         "xlwt": [],
+        "odfpy": [],
         "pytest": [],
         // If using Windows with python 2.7 and want to build using the
         // mingw toolchain (rather than MSVC), uncomment the following line.
         // "libpython": [],
     },
-
+    "conda_channels": ["conda-forge", "defaults"],
```

Comment:
Can you see if the order here matters? I think in our enivonrment.yaml file `defaults` comes first so may want to reflect here

Reply:
I think this works.